dev.yml

```diff
@@ -0,0 +1,177 @@
+- hosts: dev
+  tasks:
+    - name: Install Java package
+      sudo: true
+      yum:
+        name: java-1.8.0-openjdk
+        state: present
```

Comment:
Would it be convenient to use style like below. It'll shorten play file for repo based packages:

- name: Install essential dev packages
  sudo: yes
  yum: name={{ item }} state=installed
  with_items:
     - maven
     - gcc
     - ruby
     - .....